js/level1.js

```diff
@@ -0,0 +1,30 @@
+//layout for blockage
```

Comment:
Without file splitting and linking to particular html file, I believe you can generate levels with javascript. :)